multisig/interaction/snippets.sh

```diff
@@ -255,9 +255,9 @@ setLocalRolesMultiTransferEsdt() {
     --send --proxy=${PROXY} --chain=${CHAIN_ID}
 }
 
-setLocalRolesWrappedBridgedUSDC() {
-    getWrappedBridgedUSDCAddress
-    bech32ToHex ${getWrappedBridgedUSDCAddress}
+setLocalRolesBridgedTokensWrapper() {
+    getBridgedTokensWrapperAddress
+    bech32ToHex ${getBridgedTokensWrapperAddress}
 
     local LOCAL_BURN_ROLE=0x45534454526f6c654c6f63616c4275726e # "ESDTRoleLocalBurn"
```

Comment:
You can use this convertor from string to hex:
`"0x$(echo -n $2 | xxd -p -u | tr -d '\n')"`